app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.where(email_address: params[:email_address]).first
```

Comment:
You should use `find_by` instead of `where(...).first`.

Homework: Find out why `find_by` is better fit, you can also dig deeper in lesson 1 and read notes.